index.php

```diff
@@ -125,5 +84,72 @@ function isValidURL($url){
     
 <script src="https://code.jquery.com/jquery-3.2.1.min.js"></script>
 <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+<script type="text/javascript" charset="utf-8">
+;(function($) {
+  $(document).ready(function(){
+      var spinner = toggleSpinner();
+      $('form').submit(function(e){
+        e.preventDefault();
+        spinner();
+        $.ajax({
+          url: '/api/image-parser.php?url=' + $('input[name="url"]').val(),
+          // type : "POST",
+          dataType : 'json',
+          success : function(result) {
+            spinner();
+            if (result.success) {
+              renderStats(result);
+              renderImages(result);
+            } else {
+              $('#result').html('Invalid URL!');
+            }
+          },
+          error: function(xhr, resp, text) {
+            console.log('error', xhr, resp, text);
```

Comment:
Some error message must be displayed to the user as well

Reply:
Now there are no error message that catch this. Choose the error message, and I'll add on ajax error! ;-)

Reply:
"Could not connect to server, please try again later"